docs/details/dashboards/dashboard-home.md

```diff
@@ -1,28 +1,24 @@
 # Home Dashboard
```

Comment:
Should we have separate menu item on a left side for Home Dashboard? It is really hard to find currently.

Also where is `PMM_Home_Dashboard_Numbered.png` picture is used ?

Reply:
It is used in UI components. I will work on TOC that time will have a separate item on left side